pkg/api/volume_resp.go

```diff
@@ -82,3 +82,8 @@ type VolumeExpandResp VolumeInfo
 
 // VolumeListResp is the response sent for a volume list request.
 type VolumeListResp []VolumeGetResp
+
```

Comment:
How about this ?

```go
 type VolumeOption struct {
 	Name        string `json:"name"`
 	OnValue     string `json:"onvalue"`
 	OffValue    string `json:"offvalue"`
 }

type GroupOption struct {
        Name         string       `json:"name"`
        Options     []VolumeOption `json:"options"`
}

type GroupOptionReq GroupOption

type GroupOptionListResp []GroupOption

Reply:
s/GroupOption/OptionGroup/ sounds better.